src/services/tech-record-document-conversion.ts

```diff
@@ -241,6 +243,24 @@ const upsertVehicle = async (
   connection: Connection,
   techRecordDocument: TechRecordDocument,
 ): Promise<number> => {
+
+  if (techRecordDocument.techRecord && techRecordDocument.techRecord[0].statusCode === 'archived') {
```

Comment:
There's an enum in this repo for the status code, would be better to use that

Reply:
I can't see the enum but I can make as of the StatusCode if you would like?

`  if (techRecordDocument.techRecord && techRecordDocument.techRecord[0].statusCode === 'archived' as StatusCode) {`

Reply:
Added above as requested.